jscomp/core/js_of_lam_module.ml

```diff
@@ -29,8 +29,8 @@
 
 module E = Js_exp_make 
 
-let make ?comment (args : J.expression list) =
+let make ?comment names (args : J.expression list) =
   E.make_block
     ?comment E.zero_int_literal
-    (Blk_module None) args Immutable
+    (Blk_module (Some names)) args Immutable
 
```

Comment:
In this case `Blk_module (string list option)` the signature should be changed to `Blk_module string list`, `Blk_module None` seems an error now

Reply:
Indeed. I'd postponed this change. Done now.